crates/noseyparker/data/default/builtin/rules/thingsboard.yml

```diff
@@ -0,0 +1,24 @@
+rules:
+
+# FIXME: add MQTT rule(s)
+# - https://thingsboard.io/docs/paas/reference/mqtt-api/
+# - https://thingsboard.io/docs/reference/mqtt-api/
+
+- name: ThingsBoard Access Token
+  id: np.thingsboard.1
+
+  pattern: |
+    (?x)
+    thingsboard\.cloud/api/v1/
+    ([a-zA-Z0-9._-~]{1,32})
+
+  examples:
+  # the default access token follows the format [a-z0-9]{20} but the user can replace it with arbitrary tokens provided they are between 1 and 32 characters in length
+  # not sure if there are any other restrictions so I included all the unreserved characters that can be used in a URL without needing to be encoded
+  - http://thingsboard.cloud/api/v1/354u1g321kcqc1oad3w7/telemetry
```

Comment:
Thingsboard puts the API token in the path?? That's an interesting design choice...

Reply:
Right? It is a unique approach for sure.